### PR TITLE
Try to fix up root hashing by using map[string]interface{}

### PR DESCRIPTION
**Pull request by Martin2112**
Try to fix up root hashing by using map[string]interface{}.

### CODE REVIEW

File: crypto/data_formats_test.go [hunk 1]
```diff
@@ -15,11 +15,26 @@
...
 
 import (
+	"encoding/hex"
 	"testing"
 
 	"github.com/google/trillian"
 )
 
+// It's important that signatures don't change.
+var expectedSigHex = "5e6baba8dc3465de9c01d669059dda590b7ce123d6ccd436bcd898f1c79ff6d9"
```
Comment (codingllama) [inline crypto/data_formats_test.go]: nit: Move inside the test? It's not used anywhere else.

Comment (Martin2112) [reply crypto/data_formats_test.go]: It looked quite ugly.

Comment (Martin2112) [reply crypto/data_formats_test.go]: Oh move the var, sorry. Misunderstood.

Comment (Martin2112) [reply crypto/data_formats_test.go]: Done.

**Pull request merged**